src/Common/XMLUtils.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <Poco/DOM/DOMParser.h>
+#include <Poco/DOM/Node.h>
+#include <Poco/AutoPtr.h>
+#include <base/types.h>
+
+namespace DB:: XMLUtils
+{
+Poco::XML::Node * getRootNode(Poco::XML::Document * document);
+
+std::string getString(const Poco::XML::Node * node, const std::string & path, const std::optional<std::string> & default_value = std::nullopt);
+
+Int64 getInt64(const Poco::XML::Node * node, const std::string & path, const std::optional<Int64> & default_value = std::nullopt);
```

Comment:
I'd prefer to have two function here and in other places:

```
Int64 getInt64(const Poco::XML::Node * node, const std::string & path);
Int64 getInt64(const Poco::XML::Node * node, const std::string & path, Int64 default_value);
```

The parameter `std::optional<Int64> & default_value = std::nullopt` is hard to ready.